helm-chart/kuberay-operator/README.md

```diff
@@ -35,6 +35,16 @@ helm version
   helm install kuberay-operator .
   ```
 
+* Install KubeRay operator without installing CRDs: Use Helm's built-in `--skip-crds` flag. See [this document](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/) for more details.
```

Comment:
Is it possible to add a brief summary of why you might want to do this? I actually couldn't figure it out from the linked doc either.

The previous two bullet points (install stable version, install nightly version) are more self-explanatory, so they don't need a summary

Reply:
You should probably also mention how to install _just the CRDs_

Reply:
Good point. I will update it.